docs/application/dotnet/tutorials/library/library-list.md

```diff
@@ -0,0 +1,57 @@
+
+# Third Party Libraries for Tizen .NET Application
+
+In this list, you can find the libraries organized by purpose.
```

Comment:
done. 

Q) Are the libraries filtered based on the purpose of use as mentioned in one of the document above?
A) Yes.

Reply:
Q) Are the libraries filtered based on the purpose of use as mentioned in one of the document above?
A) Yes.
If so, then it will be good to explain it that way. Need your opinion.

Reply:
I apply your sentence.